deploy/csi-azuredisk-node.yaml

```diff
@@ -88,9 +88,6 @@ spec:
             - containerPort: 29603
               name: healthz
               protocol: TCP
-            - containerPort: 29605
```

Comment:
Same regarding the deploy/v2.0.0-alpha.1 folder?

Reply:
Added